docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,81 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low.
+
+If any of the nodes in the service exceeds the critical threshold of disk usage, the access control list (ACL) that is used to authorize API requests by Apache Kafka clients is updated on all nodes to prevent operations that would increase disk usage. Such operations include:
```

Comment:
If any of the nodes in the service exceeds the critical threshold of disk usage, the access control list (ACL)**, used to authorize API requests by Apache Kafka clients,** is updated on all nodes to prevent operations that would increase disk usage

Reply:
remove the "that is"